platform-server-kernel/src/main/java/uno/cod/platform/server/core/dto/challenge/ChallengeCreateDto.java

```diff
@@ -10,6 +10,9 @@
     private String name;
 
     @NotNull
+    private Long organizationId;
+
```

Comment:
Matches with my interpretation. Taska already have that "public" flag for "shareability", so this flag here is about whether people can participate without invitation. And as that's not super important, I'll just file an issue and we can go on with that PR.

Challenges are invite-only by default.